deep-learning/_posts/2017-02-27-unet.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: review
+title: U-Net Convolutional Networks for Biomedical Image Segmentation
+tags: deep-learning CNN segmentation medical essentials
+cite:
+  - Unet 
+  - https://arxiv.org/pdf/1505.04597.pdf
+---
+
+<style>
+  main.page-content p {
+    text-align: justify; /* helps the reading flow */
+  }
+</style>
+
+
+O. Ronneberger, P. Fischer, T. Brox **U-Net: Convolutional Networks for Biomedical Image Segmentation**, in proc of MICCAI 2015, p.234-241.
+
+[https://arxiv.org/pdf/1505.04597.pdf](https://arxiv.org/pdf/1505.04597.pdf)
```

Comment:
I have changed this a bit. You should replace the `cite` statement with this one:

```
cite:
    authors: "O. Ronneberger, P. Fischer, T. Brox"
    title:   "U-Net: Convolutional Networks for Biomedical Image Segmentation"
    venue:   "Proceedings of MICCAI 2015, p.234-241"
```

And also add the `pdf` statement:

```
pdf:  https://arxiv.org/pdf/1505.04597.pdf
```

With these changes, you can remove the reference and the link from the text. It will look like this (after merging branch `batch-norm`): 

![capture du 2017-02-28 11 08 47](https://cloud.githubusercontent.com/assets/13444373/23413461/bbf6e918-fda6-11e6-8485-97a0a9ae92f3.png)